sys/ztimer/core.c

```diff
@@ -168,6 +169,7 @@ ztimer_now_t _ztimer_now_extend(ztimer_clock_t *clock)
     clock->lower_last = lower_now;
     DEBUG("ztimer_now() returning %" PRIu32 "\n", (uint32_t)clock->checkpoint);
     ztimer_now_t now = clock->checkpoint;
+
```

Comment:
Can you at least put these unrelated whitespace additions in a separate commit? :)

Reply:
sorry, amended removal of those extra lines.